### PR TITLE
Remove unused imports in layout

### LINKED ISSUE #9346: Remove unused imports in layout
There are unused imports in some of the modules living inside `components/layout`. We should remove them...

### PR DESCRIPTION
**Pull request by kishorbhat**
Fixes #9346.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9349)

### CODE REVIEW

File: components/layout/context.rs [hunk 1]
```diff
@@ -22,7 +22,7 @@ use std::cell::{RefCell, RefMut};
 use std::collections::HashMap;
 use std::collections::hash_state::DefaultState;
 use std::rc::Rc;
-use std::sync::mpsc::{Sender, channel};
+use std::sync::mpsc::{Sender};
```
Comment (Wafflespeanut) [inline components/layout/context.rs]: This could be just `use std::sync::mpsc::Sender;`

**Comment by SimonSapin** [comment]
Thanks!

@bors-servo r+

File: components/layout/context.rs (hunk 1 above)
Comment (therealkbhat) [inline components/layout/context.rs]: Ah! Fixed.

**Pull request merged**